hbase/stackable/patches/apply_patches.sh

```diff
@@ -16,7 +16,8 @@ PATCH_DIR="patches/$VERSION"
 # Check if version-specific patches directory exists
 if [ ! -d "$PATCH_DIR" ]; then
   echo "Patches directory '$PATCH_DIR' does not exist."
-  exit 1
+  # maybe not all versions need patches
+  exit 0
```

Comment:
I know it's not perfect but in other products we do this by having an empty directory.
Can we do the same here? IMO it's more important to be consistent even if the solution ain't great

Reply:
Fixed https://github.com/stackabletech/docker-images/pull/701/commits/0afda96d8ab0f590d0c825e23972bab47181bd77